reccmp/isledecomp/compare/core.py

```diff
@@ -174,7 +181,7 @@ def _load_cvdump(self):
                     # Kicking the can down the road for now.
                     if "\x00" in decoded_string and rstrip_string == "":
                         continue
-                    sym.friendly_name = rstrip_string
+                    sym.friendly_name = decoded_string
```

Comment:
What problem did you run into here? I now see a diff on `BETA10` for strings without a code annotation. The recomp side has `\x00` on the string where orig does not. Should this be `rstrip_string` to remove those?

Reply:
I saw an error here but 2bc2a068cfa33098699bf838217a4b02114a3eb0 made it go away. I'll revert this change.